<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>seev::StartPathConfWidget</class>
 <widget class="QDialog" name="seev::StartPathConfWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Start Paths</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QListWidget" name="startPathLst"/>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="eraseBut">
     <property name="text">
      <string>Erase</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="addBut">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
